### ISSUE TITLE
It's blocking all numbers

### ISSUE THREAD

**Issue by Mo7mud** [alis-andriescu/BlockIT#1]
Hi @alis-andriescu ,
Thank you for this project.

I tried it and it's blocking all numbers not only added numbers.

Is this intended or it's a bug?

**Comment by alis-andriescu** [comment]
Hi.
Thank you for trying out this app.
Yes, it is intended to block all numbers that are not saved in Contacts :)

**Comment by Mo7mud** [comment]
Oh, I thought it will block only numbers I save in the app.
Thank you.

**Comment by alis-andriescu** [comment]
You're welcome. 
I will change the description so it will be clear what exactly this app does. 
You can also modify it so it will block only numbers that you save in the app :)

**Comment by Mo7mud** [comment]
Yes sure I'll modify it but I though it might be a bug so I wanted to report it to help you make it better.

Anyway it's very nice work.

What is the purpose of this app? Is it some kind of tutorial or college project?

**Comment by alis-andriescu** [comment]
It was a college project I made back in January and I thought maybe it will be useful to someone else, that is why I posted it on GitHub.

**Comment by Mo7mud** [comment]
Actually it was very useful for me.
I was struggling for couple of days now on how to implement call blocking.
I tried many approaches but none of them worked for me, finally your app solved my issue.

So thanks a lot you saved my day :)

**Comment by alis-andriescu** [comment]
OK :)
